### PR TITLE
Upgrade to Python 3.8

### PR DESCRIPTION
**Pull request by davidfischer**
There was more to this than I expected but it wasn't too bad.

### CODE REVIEW

File: adserver/management/commands/rtdimport.py [hunk 1]
```diff
@@ -296,6 +296,27 @@ def import_advertisements(self, advertisements_data):
             "small",
         ]
 
+        with open(
```
Comment (ericholscher) [inline adserver/management/commands/rtdimport.py:299]: I assume this was automated? We can probably just delete this file.

File: docker-compose/local/django/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.6-alpine
+FROM python:3.8-alpine
```
Comment (ericholscher) [inline docker-compose/local/django/Dockerfile:1]: I'd love to move this to Ubuntu to match production at some point, if it isn't hard to do.

**Review by ericholscher — APPROVED** [review]
Looks good :+1:

File: adserver/management/commands/rtdimport.py (hunk 1 above)
Comment (davidfischer) [reply adserver/management/commands/rtdimport.py:299]: I agree this file can be deleted. We're never running it again. I'll do a separate PR after merging this.

File: docker-compose/local/django/Dockerfile (hunk 2 above)
Comment (davidfischer) [reply docker-compose/local/django/Dockerfile:1]: It shouldn't be that hard.

**Review by stsewd — APPROVED** [review]

**Pull request merged**